extras/gha_build.sh

```diff
@@ -0,0 +1,352 @@
+#!/bin/bash
+
+# gha_build.sh
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# This shell script is part of the NEST Travis CI build and test environment.
+# It is invoked by the top-level Travis script '.travis.yml'.
+#
+# NOTE: This shell script is tightly coupled to 'extras/parse_travis_log.py'.
+#       Any changes to message numbers (MSGBLDnnnn) or the variable name
+#      'file_names' have effects on the build/test-log parsing process.
+
+
+# Exit shell if any subcommand or pipline returns a non-zero status.
+set -ex
+
+env
+if [ "$xNEST_BUILD_COMPILER" = "CLANG" ]; then
+    export CC=clang-7
```

Comment:
clang-7 seems rather old, the current version is 11. Is there a specific reason to use clang-7?

Reply:
No specific reason. Thanks for pointing this out. I just added a clang-11 build.